config.yaml.example

```diff
@@ -0,0 +1,33 @@
+chain_name: "Starknet Mainnet"
```

Comment:
Could you drop some documentation describing each of these fields? Also you should rename it  `config.yaml.exemple`

Reply:
Done